### PR TITLE
Move datasets into model registry W&B

### PR DESCRIPTION
**Pull request by sarzhann**
* All files from `data/` are moved into https://wandb.ai/clarify/climate-futures-clarify/artifacts/dataset/sea_temperature/v0
* Implemented model registry manager, which can pull/push datasets from W&B
* The only requirement is to set the `WANDB_TOKEN` env variable before execution.

### CODE REVIEW

**Comment by sarzhann** [comment]
@olewu you can consider this change as a helper utility for moving datasets outside of the package into W&B.
So, it can be used inside your work, when the training step is automated and can be done on our side.

File: src/forecast_clarify/model_registry.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import wandb
```
Comment (olewu) [inline src/forecast_clarify/model_registry.py:1]: do we need to include this in the requirements?

Comment (sarzhann) [reply src/forecast_clarify/model_registry.py:1]: great catch, I'll add it

Comment (sarzhann) [reply src/forecast_clarify/model_registry.py:1]: fixed.

File: requirements.txt [hunk 2]
```diff
@@ -9,3 +9,4 @@ numpy==1.21.0 #1.23.0
 pandas==1.3.5 #1.4.3
 scikit-learn==1.0.2 #1.1.2
 xarray==0.20.2 #2022.3.0
```
Comment (olewu) [inline requirements.txt:11]: I noticed that when I install the requirements only, I cannot load the datasets (e.g. in the example notebook). The issue seems to be that xarray is missing a backend for the netcdf files. I can fix this with `pip install netCDF4`. Do you run into the same issue? If so, maybe we have to add netCDF4 to the requirements.

File: tests/test_config.py [hunk 3]
```diff
@@ -1,9 +1,17 @@
-from forecast_clarify.config import dirs
+from forecast_clarify.src.forecast_clarify.model_registry import Manager
```
Comment (olewu) [inline tests/test_config.py:1]: does this work with just `from forecast_clarify.model_registry import Manager`?

File: requirements.txt (hunk 2 above)
Comment (sarzhann) [reply requirements.txt:11]: I didn't see that issue, but probably it's fine to have `netCDF4` in the requirements.txt as well

**Review by olewu — APPROVED** [review]

**Pull request merged**